src/include/duckdb/planner/logical_operator.hpp

```diff
@@ -77,6 +77,8 @@ class LogicalOperator {
 
 	static unique_ptr<LogicalOperator> Deserialize(Deserializer &deserializer, PlanDeserializationState &state);
 
+	static unique_ptr<LogicalOperator> Copy(unique_ptr<LogicalOperator> &op, ClientContext &context);
```

Comment:
Could we define this to be more consistent with all the other Copy functions in the system? i.e.:

```sql
virtual unique_ptr<LogicalOperator> Copy() const = 0;
```

Reply:
Done. Thanks!